include/crm/common/schemas.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2024 the Pacemaker project contributors
+ *
+ * The version control history for this file may have further details.
+ *
+ * This source code is licensed under the GNU Lesser General Public License
+ * version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+ */
+
+#ifndef PCMK__CRM_COMMON_SCHEMAS__H
+#  define PCMK__CRM_COMMON_SCHEMAS__H
+
+#include <stdbool.h>        // bool
+#include <libxml/tree.h>    // xmlNode
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*!
+ * \file
+ * \brief XML schema API
+ * \ingroup core
+ */
+
+bool pcmk_update_configured_schema(xmlNode **xml, bool to_logs);
```

Comment:
Since we're making this public, might as well return a standard code so that it's extensible (however unlikely extending it may be)

Reply:
good idea